pkg/cmd/auth/refresh/refresh.go

```diff
@@ -0,0 +1,127 @@
+package refresh
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/MakeNowJust/heredoc"
+	"github.com/cli/cli/internal/config"
+	"github.com/cli/cli/pkg/cmdutil"
+	"github.com/cli/cli/pkg/iostreams"
+	"github.com/cli/cli/pkg/prompt"
+	"github.com/spf13/cobra"
+)
+
+type RefreshOptions struct {
+	IO     *iostreams.IOStreams
+	Config func() (config.Config, error)
+
+	Hostname string
+	Scopes   []string
+}
+
+func NewCmdRefresh(f *cmdutil.Factory, runF func(*RefreshOptions) error) *cobra.Command {
+	opts := &RefreshOptions{
+		IO:     f.IOStreams,
+		Config: f.Config,
+	}
+
+	cmd := &cobra.Command{
+		Use:   "refresh",
+		Args:  cobra.ExactArgs(0),
+		Short: "Request new scopes for a token",
+		Long: heredoc.Doc(`Expand the permission scopes for a given host's token.
```

Comment:
I realize I'm being super nitpicky, but permissions scopes typically can't be added to an OAuth token (they can be added to PATs, but that needs to be done manually by the user in the web UI). A new token is requested instead.

Furthermore, from the perspective of the user, we're merely refreshing the stored credentials for GitHub CLI and, unless the user has pasted in a token via `--with-token`, the user does not necessarily have to be aware that these credentials involve something called a "token". That might be an implementation detail at this point.

I suggest that we use a more general-sounding phrase such as "stored credentials" when talking about authentication information stored in `hosts.yml`.

Reply:
I took the word "token" out of all the docs for this command.